packages/expo-video/src/VideoPlayer.web.tsx

```diff
@@ -138,6 +140,24 @@ export default class VideoPlayerWeb
     this._preservesPitch = value;
   }
 
+  get timeUpdateEventInterval(): number {
+    return this._timeUpdateEventInterval;
+  }
+  set timeUpdateEventInterval(value: number) {
+    this._timeUpdateEventInterval = value;
+    if (this._timeUpdateLoop) {
+      clearInterval(this._timeUpdateLoop);
+    }
+    if (value > 0) {
+      // Emit the first event immediately like on other platforms
+      this.emit('timeUpdate', { currentTime: this.currentTime });
```

Comment:
I wonder if we should explicitly pass `currentLiveTimestamp` and `currentOffsetFromLive` as `null`s?